src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,335 @@
+template.
+
+We will use existing `ActionFactory` from [knotx-fragments](https://github.com/Knotx/knotx-fragments) to achieve our goal
+which will be implementing custom endpoint with cache ability.
+
+## Prerequisites
+- JDK 8
+- Docker
+
+## What you're going to learn:
+- How to add cache ability to API
+- How to set up Wiremock (we don't need to implement real API, mock API will be enough for our training purposes)
+- How to deploy project to Docker
+
+Download the [Latest Knot.x Starter Kit release](https://github.com/Knotx/knotx-starter-kit/releases) and unzip it.
+
+Project has the following structure:
+```
+├── docker
+|   ├── Dockerfile                // Docker file with image definition.
+├── functional                    // Keep here your functional tests. Example implementation included
+├── gradle                        // Gradle wrapper and common gradle scripts
+├── knotx/conf                    // Knotx configuration which will be copied to docker image
+├── modules                       // Sub-modules of your project
+│   ├── ...                       // example modules implementation
+```
+
+## Project configuration
+The first thing we need to do is to specify configuration files. Let's start in the project level directory and simply create `api-cache.yml`
+The file should contains
+``` yml
+version: '3.7'
+
+networks:
+  knotnet:
+
+services:
+
+  webapi:
+    image: rodolpheche/wiremock
+    volumes:
+      - "./common-services/webapi:/home/wiremock"
+    ports:
+      - "3000:8080"
+    networks:
+      - knotnet
+
+  knotx:
+    image: knotx-example/api-cache:latest
+    command: ["knotx", "run-knotx"]
+    ports:
+      - "8092:8092"
+      - "18092:18092"
+    networks:
+      - knotnet
+```
+
+First and foremost it specifies docker virtual network in which our services will be deployed. Then we have some service configs.
+The first one is `webapi`, `image` - docker image is provided here, `volumes` - directories to which Docker will have access, 
+in our case it's directory where all Wiremock configs are stored,
+`ports` - on the left side is provided port accessible from `localhost`, and on the right side we have port number available in 
+virtual environment. The last param `networks` specifies in which docker network service will be deployed.
+Then we have `knotx` service specification, it's very similar to previous one. `command` property is used to point Knot.x starting commands.
+
+Next step in our configuration is to properly set up `settings.gradle.kts` (also at the project level directory) 
+```gradle
+rootProject.name = "api-cache"
+
+include("health-check")
+
+project(":health-check").projectDir = file("modules/health-check")
+``` 
+
+Here we have to specify project name, and packages to be included in this project. 
+
+Next thing to do is to configure `gradle.properties` at project level directory. Let's fill it with
+```gradle
```

Comment:
maybe let's just limit the config change to renaming the Docker image, that way it will be interoperable with future Knot.x versions